app/commands/xp.js

```diff
@@ -131,7 +134,47 @@ module.exports = Base => class extends Base {
         await this.simpleXpCheck(playerName);
     };
 
+    async handleXpAutoStore(playerName) {
+        //turn on or off the autostore.
+        if (await this.readPlayerXpAutoStore(playerName)) {
+            await this.xpAutoStoreDisable(playerName);
+        } else {
+            await this.xpAutoStoreEnable(playerName);
+        }
+    };
+
+    async xpAutoStoreEnable(playerName) {
```

Comment:
This and the disable function are not really much different from each other. Do you think there s a way you could join them up?